### ISSUE TITLE
No Autocomplete

### ISSUE THREAD

**Issue by pencilboy99** [atom/autocomplete-plus#620]
I'm not getting autocomplete behavior either automatically or with control space. I have all the defaults set and have deactivated vim-mode

Using latest atom on windows.

**Comment by mnquintana** [comment]
What language grammar do you have active? Can you reproduce this in safe mode? (`atom --safe`)

**Comment by pencilboy99** [comment]
Okay, I get it. I did get it to work - I opened a *.js file and it did autocomplete. I was editing a .txt and a .bat file. How do I tell it (configure it) so that if it is using filetype N (e.g., .txt) it does word completion (searches the buffer for any matching word). Thanks!

**Comment by pencilboy99** [comment]
Is there a way to get it to do buffer complete (i.e., whatever the words in the buffer are also with them?

**Comment by lee-dohm** [comment]
@pencilboy99, yes, you can go into the autocomplete-plus package settings and check "Enable Built-In Provider". This will do the "buffer complete" that you're asking for in any file type that isn't in the "Built-In Provider Blacklist".

I see that your original issue is covered by #557 which you also commented on. Between the two, does this cover all your questions around autocomplete? Can we close this issue?
